integration-libs/s4-service/order/components/reschedule-service-order/reschedule-service-order.component.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, inject, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { CartOutlets } from '@spartacus/cart/base/root';
+import {
+  GlobalMessageService,
+  GlobalMessageType,
+  RoutingService,
+} from '@spartacus/core';
+import { OrderDetailsService } from '@spartacus/order/components';
+import { Order } from '@spartacus/order/root';
+import {
+  CheckoutServiceSchedulePickerService,
+  RescheduleServiceOrderFacade,
+  ServiceDateTime,
+} from '@spartacus/s4-service/root';
+import { combineLatest, map, Observable, Subject, takeUntil } from 'rxjs';
+
+@Component({
+  selector: 'cx-reschedule-service-order',
+  templateUrl: './reschedule-service-order.component.html',
+})
+export class RescheduleServiceOrderComponent implements OnInit, OnDestroy {
+  protected orderDetailsService = inject(OrderDetailsService);
+  protected rescheduleServiceOrdeFacade = inject(RescheduleServiceOrderFacade);
```

Comment:
rescheduleServiceOrderFacade correct the speling

Reply:
corrected